linters/elixir/.credo.exs

```diff
@@ -0,0 +1,21 @@
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        included: ["lib/", "src/", "web/", "apps/"],
+        excluded: [~r"/_build/", ~r"/deps/"]
+      },
+      requires: [],
+      check_for_updates: true,
+      strict: true,
+      color: true,
+
+      checks: [
+        {Credo.Check.Design.TagTODO, exit_status: 0},
+        {Credo.Check.Readability.ModuleDoc, false},
```

Comment:
That's something that Elixir tries to enforce as a good practice as well, and I like it, but for public projects, not for internal apps. I'm thinking if it would make sense to have an OSS linter and an internal linter. /cc @naps62